Symfony/CS/Fixer/PSR2/OneLineMethodArgumentFixer.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Konrad Cerny <info@konradcerny.cz>
+ */
+class OneLineMethodArgumentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound('(');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        /** @var Token $token */
+        foreach ($tokens as $index => $token) {
+            if (!$token->equals('(')) {
+                continue;
+            }
+
+            if (
+                // function
+                $this->isFunction($tokens, $index - 3)
+                // callable
+                || $this->isFunction($tokens, $tokens->getPrevMeaningfulToken($index))
+                // function call
+                || $this->isFunction($tokens, $tokens->getPrevMeaningfulToken($index), T_STRING)
```

Comment:
The "function" condition does not match this function:

```php
function /*comment*/ foo (...
```

But all function declarations (except callables) are matched by the "function call" condition, because there is also a `T_STRING` at the prev meaningful position (`foo`).

So the first condition is useless.